### ISSUE TITLE
error: method does not override or implement a method from a supertype @Override

### ISSUE THREAD

**Issue by Amurmurmur** [react-native-community/react-native-blur#219]
Starts to happen when upgraded to RN 0.47.0-rc.0, when comping on android:

```
:react-native-blur:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/Users/amurmurmur/exampleapp/node_modules/react-native-blur/android/src/main/java/com/cmcewen/blurview/BlurViewPackage.java:23: error: method does not override or implement a method from a supertype
    @Override
    ^
Note: /Users/amurmurmur/exampleapp/node_modules/react-native-blur/android/src/main/java/com/cmcewen/blurview/BlurringView.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
:react-native-blur:compileReleaseJavaWithJavac FAILED
```

How can it be fixed? 
Thank you in advance

**Comment by TrustTheBoy** [comment]
[https://github.com/react-native-community/react-native-blur/pull/226](look here)

**Comment by PeterEmanuel** [comment]
Does this mean that the packages/projects that I link in my app have to be fixed for this?
Not funny, if this is true...

**Comment by Kureev** [comment]
Fixed in `3.1.3`
